apps/web/components/walkthrough/Walkthrough.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --walkthrough-side-nav-width: 0;
+}
+
+.web-Walkthrough {
+  height: calc(100vh - var(--header-height));
+  overflow: hidden;
+  display: grid;
+}
+
+.web-Walkthrough--SideNav {
+  display: none;
+  border-right: var(--layout-border-width-small) solid
+    var(--surface-color-border-default);
+}
+
+@media (min-width: 38rem) {
+  :root {
+    --walkthrough-side-nav-width: 15.625rem; /* 250px */
+  }
+
+  .web-Walkthrough {
+    grid-template-columns: var(--walkthrough-side-nav-width) 1fr;
+  }
+
+  .web-Walkthrough--SideNav {
+    display: block;
+  }
+}
+
+/* DESKTOP - 912px */
+@media (min-width: 57rem) {
```

Comment:
Should we explore adding a responsive.css file that houses media-queries definitions and then we can reference those without needing to annotate?

Reply:
Sadly we can't. The one drawback of vanilla CSS currently is the inability to use custom properties in media queries. The ONE thing I don't like about this setup. This is why I've been trying to keep the media queries to the variables.css file as much as possible with just updating the custom properties there. I just committed a refactoring that does this for a few cases here.